apps/website/src/examples/Radio.main.tsx

```diff
@@ -8,8 +8,10 @@ import { Flex, Radio } from '@itwin/itwinui-react';
 export default () => {
   return (
     <Flex flexDirection='column' alignItems='flex-start'>
-      <Radio label='Apple' value='apple' name='fruit' />
-      <Radio label='Orange' value='orange' name='fruit' />
+      <Radio label='Option 1' defaultChecked />
+      <Radio label='Option 2' />
+      <Radio label='Option 3' defaultChecked disabled />
+      <Radio label='Option 4' disabled />
```

Comment:
```suggestion
      <Radio name='Options' label='Option 1' defaultChecked />
      <Radio name='Options' label='Option 2' />
      <Radio name='Options' label='Option 3' disabled />
```
A few changes suggested here:
1. Because radios can only have 1 checked item, we can delete line 13 because line 11 is already set active.
2. Added `name` so that these 3 radios are treated as a group, forcing only 1 radio to be checked. This was not documented previously, so is a nice catch & inclusion for this demo!